js/components/ReportSummaryForm.vue

```diff
@@ -5,40 +5,60 @@
       <ul v-if="hasErrors">
         <li v-for="error in errors" :key="error" class="text-danger font-weight-bold">{{ error }}</li>
       </ul>
-      <div class="form-group">
-        <label>Title <input id="title" name="title" v-model="title" type="text" class="form-control"></label>
+      <div class="row">
```

Comment:
Any reason why you removed the form-group class? I think this is what makes all of the form-controls full width.